pgml-dashboard/src/main.rs

```diff
@@ -95,5 +109,79 @@ mod test {
         let response = client.get("/dashboard/snapshots/").dispatch().await;
         assert_eq!(response.status().code, 200);
     }
-}
 
+    #[rocket::async_test]
+    async fn test_snapshot_entries() {
+        let snapshots_endpoint = "/dashboard/snapshots/";
+        let client = Client::tracked(rocket().await).await.unwrap();
+        let response = client.get(snapshots_endpoint).dispatch().await;
+
+        let body = response.into_string().await.unwrap();
+        let snapshot_links = get_href_links(body.as_str(), snapshots_endpoint);
```

Comment:
This will follow the links in the menu I think, so it'll double test those pages, which is totally fine since it's fast.

Reply:
All those links are dropped in `get_href_links` function here 

```
if href.contains(pattern) && href != pattern {
                output.push(String::from(href));
}
```